source/convertSchemaToInterface.ts

```diff
@@ -166,7 +167,8 @@ function convertTypeToTsType(key: string, value?: TypedSchemaProperty) {
     if (value.items.$ref) {
       return `${value.items.$ref}[]`;
     } else if ("type" in value.items && value.items.type) {
-      return `${convertTypeToTsType(value.items.type as string)}[]`;
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
This was a bit weird...

In the call below, the second argument wasn't specified, and hence would default to `undefined`. But that should crash the recursive call, since we (on line 156 check for the type of the value).

So I actually think we have a runtime bug here, but not sure how to fix it, or how to test it. Not sure what the intended behavior is.

If you can tell me, I can update accordingly, add a test, and fix the bug.

Reply:
Mistake on my part. That never happens. There are properties that are named `items`, and I must have read the name of the property instead of the `items` key inside that property and confused myself.

Reply:
So do I just remove the whole `else if`?

Reply:
Yes

Reply:
Great that the tests are set up, because there shouldn't be any difference :)

Reply:
Haha yeah wow didn't even think about that 🤣 the full schema snapshot test should be able to verify it. Thanks!

Reply:
This has been fixed now.